src/infrastructure/jira/jira-service.ts

```diff
@@ -75,11 +75,15 @@ export class JiraService {
 		connectInstallation: ConnectInstallation,
 	): Promise<void> => {
 		const associationsLastUpdated = new Date();
+
+		// Jira does not allow display names longer than 255 characters.
+		const MAX_DISPLAY_NAME_LENGTH = 255;
 		const response = await jiraClient.submitDesigns(
 			{
 				designs: designs.map(
 					({ design, addAssociations, removeAssociations }) => ({
 						...design,
+						displayName: design.displayName.slice(0, MAX_DISPLAY_NAME_LENGTH),
```

Comment:
**suggestion (non-blocking):** Please, consider adding adding `…` to the end of the string to indicate that it was truncated.

Here how we could do this:

1. Add `truncate` function to `common/stringUtils` (inspired by [this](https://javascript.info/task/truncate)):
  ```ts
  function truncate(str, maxLength) {
    if (str.length <= maxLength) return str;
    
    return `${str.slice(0, maxlength - 1)}…`;
  }
  ```
2. Call `truncate(design.displayName, MAX_DISPLAY_NAME_LENGTH)` here.

Reply:
Good suggestion! I added the helper function and some tests.